### PR TITLE
Force consistent capitalization of Polymer/ PolymerElements and PolymerLabs in bower.json

### LINKED ISSUE #47: Consistency "PolymerElements" vs "polymerelements" in bower.json for webjars usage
This ticket was first created by me in the https://github.com/PolymerElements/app-layout/issues/293 and @abdonrd kindly ask me if I can created it in tedium as well, which seems to be a good place for it.

### Description

Many PolymerElements github projects like the `app-layout` element is using "polymerelements" in its bower.json file to reference dependecies. 

Like: `polymerelements/iron-flex-layout#^1.0.0` 

While other popular Polymer elements like `paper-checkbox`, `paper-button` use "PolymerElements"

Like `PolymerElements/iron-flex-layout#^1.0.0`

Other Polymer elements use a mixture like `app-route`

I assume this does not matter in the JavaScript and Bower world. But I'm trying to figure out if it is possible to use Polymer-Elements in Java/JVM based micro services. In the Java world dependencies are pulled together from so called "Maven-Repositories". 

There is a very nice effort to bridge the Bower/NPM things with Maven-Repositories in the project http://www.webjars.org/ . Webjars.org works fairly well for pure javascript dependencies, but less well for PolymerElements (and they have many transitive dependencies by nature). Many PolymerElements artifacts appear twice with the lower and upper case version of the github name "PolymerElements".

`github-com-PolymerElements-iron-flex-layout`
`github-com-polymerelements-iron-flex-layout`

This is an issue for JVM based software where letter case matters.
### Expected outcome

It would be nice if all PolymerElements would consistently use "PolymerElements" because this is the github name of the project.
### Steps to reproduce

Look at the bower.json files of the various projects
- https://github.com/PolymerElements/paper-checkbox/blob/master/bower.json
- https://github.com/PolymerElements/paper-button/blob/master/bower.json
- https://github.com/PolymerElements/app-layout/blob/master/bower.json
- https://github.com/PolymerElements/app-route/blob/master/bower.json
- https://github.com/PolymerElements/iron-dropdown/blob/master/bower.json
- ...
### Browsers Affected

This does not affect bowsers. This affects Java build system like Maven, Gradle, SBT but seems to be no problem for JavaScript users that use Bower.

### PR DESCRIPTION
**Pull request by rictic**
Fixes #47

/cc @abdonrd

### CODE REVIEW

File: cleanup-passes/travis.ts [hunk 1]
```diff
@@ -147,5 +147,5 @@ async function cleanupTravisConfig(element: ElementRepo): Promise<void> {
 register({
   name: 'travis',
   pass: cleanupTravisConfig,
-  runsByDefault: true,
+  runsByDefault: false,
```
Comment (dfreedm) [inline cleanup-passes/travis.ts:150]: This change seems unrelated. Intentional?

**Comment by dfreedm** [comment]
LGTM with the one nit.

**Comment by abdonrd** [comment]
LGTM!

File: cleanup-passes/travis.ts (hunk 1 above)
Comment (rictic) [inline cleanup-passes/travis.ts:150]: Intentional. We never landed all of the travis fixups, which still need some hand-fixing afaik

**Comment by rictic** [comment]
Leaving this open for now, I don't want to potentially break the world on a friday afternoon, will wait to monday to merge and run it against our elements when I'll have time to do it carefully

**Comment by abdonrd** [comment]
Good choice @rictic! :)

**Comment by bartolom** [comment]
I have also created https://github.com/Polymer/polymer/pull/3844 which is related

**Comment by bartolom** [comment]
Dear Polymer Team,

Maybe you don't have time because you focus on Polymer 2.x (which I'm looking forward) or you have some technical concerns to do this automated.

I'm really stuck with this in Polymer 1.x and would also volunteer to create manual pull requests for each of the PolymerElements subprojects that are inconsistent in their "main" dependencies. I counted and documented them in https://github.com/PolymerLabs/tedium/issues/47 and they are only 22 which is kind of doable

**Comment by stramel** [comment]
one eternity later

**Comment by stramel** [comment]
@rictic @azakus any progress on this?
